test/packages/with-custom-agent/auditd_manager_independent_agent/data_stream/auditd/_dev/test/system/test-default-config.yml

```diff
@@ -0,0 +1,13 @@
+data_stream:
+  vars:
+    audit_rules:
+      - "-a always,exit -F arch=b64 -S execve,execveat -k exec"
+    preserve_original_event: true
+agent:
+  user: "root"
+  runtime: docker
+  # shared_pid_address_space: true ?
+  pid_mode: "host"
```

Comment:
This setting looks like that it is more related to docker runtime.

It is needed to specify somehow this setting for `auditd_manager` package to keep the same behavior. That setting is present in the current configuration:
https://github.com/elastic/elastic-package/blob/052cfc8d3d0bfba8355cdea12289ee1a40fa4949/test/packages/with-custom-agent/auditd_manager/data_stream/auditd/_dev/deploy/agent/custom-agent.yml#L4-L8

Should it be added here even if it is more docker related ? or maybe do we need to add a map for docker specific settings ? 
```
agent:
  user: root
  runtime: docker
  capabilities:
    - AUDIT_CONTROL
    - AUDIT_READ
  docker:
    pid_mode: host
```

Maybe it can be changed the name of this field to `use_host_pid_namespace: true`

WDYT @jsoriano ?

Reply:
PID namespacing is not specific of docker, so I would not use an specific docker variable. Not sure about the variable name to use but I wouldn't use a boolean because there can be more than one option.